pkgs/tools/misc/grub/pvgrub_image/configs/grub.cfg

```diff
@@ -0,0 +1,9 @@
+if search -s -f /boot/grub/grub.cfg ; then
+	echo "Reading (${root})/boot/grub/grub.cfg"
```

Comment:
Why are there two paths to read grub.cfg from? One is /boot/grub/grub.cfg. If we only use one, eliminate one please.